### ISSUE TITLE
Linker document example error 

### ISSUE THREAD

**Issue by meistudioli** [ampproject/amphtml#21226]
To Whom it may concern:

Here comes typo error about linker example. Check the following document.

[Linker ID Receiving](https://github.com/ampproject/amphtml/blob/master/extensions/amp-analytics/linker-id-receiving.md#receiving-linker-params-on-amp-pages)

```
'cookies': {
  'enabled': true,
  'cookieName1': {
    'value': 'QUERY_PARAM(example)',
  },
  'cookieName2': {
    'value': 'LINKER_PARAM(exampleName, exampleValue)',
  },
}
```

It should be 

```
'cookies': {
  'enabled': true,
  'cookieName1': {
    'value': 'QUERY_PARAM(example)',
  },
  'cookieName2': {
    'value': 'LINKER_PARAM(exampleName, exampleId)',
  },
}
```

Paul

**Comment by cathyxz** [comment]
/triaging to @calebcordry I think? Feel free to reassign.

**Comment by calebcordry** [comment]
@meistudioli thanks for reporting. You are correct, naming is hard :) Would you be willing to make a pull request for this change? If so, tag me and I will review. Thanks!

**Comment by meistudioli** [comment]
Hi, @calebcordry :

Here comes the [PR](https://github.com/ampproject/amphtml/pull/21307). Review please. :)

Paul

**Comment by calebcordry** [comment]
Closed by #21307
